Realme/C2/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="me.phh.treble.overlay.realme.c2">
+    <overlay 
+	android:isStatic="true" 
+	android:priority="125" 
+	android:targetPackage="android"
+        android:requiredSystemPropertyName="ro.product.vendor.model"
+	android:requiredSystemPropertyValue="+RMX194*" />
```

Comment:
Let's hope you're right there...

Reply:
yes i tried on both rmx1941 & 1945 app is getting installed on both no parsing error